base/deprecated.jl

```diff
@@ -209,7 +209,7 @@ for deprecatedfunc in [:isprime, :primes, :primesmask, :factor]
     @eval begin
         $deprecatedfunc(args...) = error(string($deprecatedfunc, args,
             " has been moved to the package Primes.jl.\n",
-            "Run Pkg.add(\"Primes\") to install Primes on Julia v0.5-"))
+            "Run Pkg.add(\"Primes\") to install Primes on Julia v0.5- and then run `using Primes`."))
```

Comment:
What does `v0.5-` mean? I know you didn't write the original message, but I think perhaps it can be cleared up. "In Julia v0.5 and later", maybe?